### ISSUE TITLE
Support Service doc comments for OpenApi/Swagger generation

### ISSUE THREAD

**Issue by seanlaff** [grpc-ecosystem/grpc-gateway#4419]
For methods we can control the OpenAPI title/description via comments
```protobuf
// Update a book
//
// This method updates a book
 rpc UpdateBook(UpdateBookRequest) returns (Book) { 
   option (google.api.http) = { 
     patch: "/v1/{book.name=publishers/*/books/*}" 
     body: "book" 
   }; 
 } 
 ```
 However, I can't find any documentation for doing the same at the service level, and I don't see it in ALittleBitOfEverything.proto. I'd like to do
 ```protobuf
 // Book
//
// Methods of interacting with books
service BookService {
}
```
and have it function as if I did
```protobuf
option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_tag) = {
  name: "Book"
  description: "Methods of interacting with books"
}

**Comment by johanbrandhorst** [comment]
HI, thanks for your issue! I can't tell if this is a straightforward mapping, is a `openapiv2_tag` analogous to a `service`? I'd be happy to look at a PR that experiments with this idea to see if it makes sense.

**Comment by seanlaff** [comment]
Yup, that is the distinction the generator currently makes. Of note is that all the methods attached to a service need to point to the renamed `openapiv2_tag`, and not the original name, else they won't render properly in the swagger UIs. I imagine a first pass at this issue may run into the same issues as described in https://github.com/grpc-ecosystem/grpc-gateway/issues/3829#issuecomment-1994218163

**Comment by ankur-kalita** [comment]
heyy @seanlaff can you please assign this issue to me ?

**Comment by johanbrandhorst** [comment]
There's no need for assignments, if you want to work on this, go ahead.

**Comment by lhdhtrc** [comment]
I've also encountered this problem. Is there no good solution now

**Comment by lhdhtrc** [comment]
option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_tag) = {
  name: ""
}; 
The tag in the interface should be overwritten. If the tag is defined in the interface, it will overwrite the tag in the service
